war/src/main/webapp/scripts/yui/button/button-debug.js

```diff
@@ -2719,7 +2719,7 @@ version: 2.9.0
                     method as well.
                 */
               
-                if ((UA.ie || UA.webkit || UA.gecko) && bSubmitForm) {
```

Comment:
Add a comment here to reference the historical Gecko issue ?

Reply:
OK, I've added 626fa0fff3dfbf2a59b5a0682df76cb7fa99ff66 to document the Firefox (Gecko) situation.